packages/desktop-client/public/data/migrations/1679728867040_rules_conditions.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN TRANSACTION;
+
+ALTER TABLE rules ADD COLUMN conditionsOp TEXT DEFAULT 'and';
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏we try to follow snake_case in database columns (I know.. some are using camelCase.. but for the most part it's snake).

```suggestion
ALTER TABLE rules ADD COLUMN conditions_op TEXT DEFAULT 'and';
```